### PR TITLE
Draft | add celery exporter

### PR DESCRIPTION
**Pull request by santoshr1016**
## Description

Add celery exporter

## Related Issues

Do not merge this PR until this text is replaced with links to related issues.

## Testing

Do not merge this PR until this text is replaced with details about how these changes were tested.

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Comment by pgvishnuram** [comment]
Deployment for celery exporter don't seems to have condition

**Comment by danielhoherd** [comment]
Internal discussion about whether this should go into the astronomer airflow chart or OSS airflow chart <https://astronomer.slack.com/archives/CGQSYG25V/p1659105964163119>

File: values.yaml [hunk 1]
```diff
@@ -426,3 +426,39 @@ astronomer:
...
+    enabled: true
+    replicaCount: 1
+    podAnnotations: {}
+    podSecurityContext: {}
+    env:
+      - name: CE_BROKER_URL
+        valueFrom:
+          secretKeyRef:
+            name: "{{ .Release.Name }}"
+            key: connection
```
Comment (pgvishnuram) [inline values.yaml:440]: this can be removed

File: values.yaml [hunk 2]
```diff
@@ -426,3 +426,39 @@ astronomer:
...
+      portName: celery-exporter
+#      labels:
+#        {}
+#        # prometheus.io/path: /metrics
+#        # prometheus.io/port: "9121"
+#        # prometheus.io/scrape: "true"
+#    redisAddressConfig:
+#      enabled: false
+#    auth:
+#      enabled: false
```
Comment (pgvishnuram) [inline values.yaml:464]: this can also be removed

File: templates/celery-exporter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#################################################
+## Deployment                                ####
+#################################################
```
Comment (pgvishnuram) [inline templates/celery-exporter/templates/deployment.yaml:3]: Celery Exporter Deployment

File: templates/celery-exporter/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#################################################
+## Deployment                                ####
+#################################################
```
Comment (pgvishnuram) [inline templates/celery-exporter/templates/service.yaml:3]: Celery Exporter Service

File: templates/celery-exporter/templates/serviceaccount.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#################################################
+## Deployment                                ####
+#################################################
```
Comment (pgvishnuram) [inline templates/celery-exporter/templates/serviceaccount.yaml:3]: Celery Exporter ServiceAccount

**Pull request closed without merging**